src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -3260,6 +3280,19 @@ private void CreateContainerProcessInternal()
             ErrorCode = error;
         }
 
+        /// <summary>
+        /// Get Command to launch container process based on instance properties.
+        /// </summary>
+        private string GetContainerProcessCommand()
+        {
+            return string.Format(System.Globalization.CultureInfo.InvariantCulture,
```

Comment:
Using string interpolation and [FormattableString.Invariant](https://docs.microsoft.com/en-us/dotnet/api/system.formattablestring.invariant?view=netcore-2.1#System_FormattableString_Invariant_System_FormattableString_) would make this string construction easier to read:
```csharp
using static System.FormattableString;
private string foo()
{
   return Invariant($"bar {expression} {baz == null ? 1 : 0}");
}
```